src/signals/incident/components/form/WithHeading/WithHeading.tsx

```diff
@@ -18,6 +18,10 @@ type WithHeadingProps = FormOptions & {
 const StyledHeading = styled(Heading)`
   font-weight: 500;
   margin: ${themeSpacing(2, 0, 5)};
+
+  @media ${breakpoint('max-width', 'tabletS')} {
+    font-size: 20px;
+  }
```

Comment:
Looked it up, and we use screen because it is more compatibel with accessibility tools. 

```suggestion
  @media screen and ${breakpoint('max-width', 'tabletS')} {
    font-size: 20px;
  }
```

Reply:
Ha Vincent, ik heb het nog even aangepast met dat screen. Wil je het nog weer bekijken of zal ik squash&mergen?

Reply:
Squash and merge!